src/views/Change.vue

```diff
@@ -219,6 +230,18 @@ export default class Change extends Mixins(
         ).catch(() => cloneDeep(EmptyFees))
       )
 
+      // set current profile name to store for field pre population
+      // do this only if we are not staff
+      if (!this.isRoleStaff) {
+        // pre-populate Certified By name
+        this.setCertifyState(
+          {
+            valid: this.getCertifyState.valid,
+            certifiedBy: `${this.getUserFirstName} ${this.getUserLastName}`
+          }
+        )
+      }
```

Comment:
OK, let's not let this hold up this PR.

Could you ask her again at the next stand-up meeting? If this ticket is still open then use it, otherwise we can create a new ticket.

Reply:
Yes I will she never got back to me, she is probably to busy.